server/views/temporary-accommodation/assessments/summary.njk

```diff
@@ -33,9 +33,11 @@
         <div class="moj-identity-bar__container">
             <div class="moj-identity-bar__details">
         <span class="moj-identity-bar__title">
-          <span class="govuk-caption-xl">Referral summary</span>
-          <h1 class="govuk-heading-xl">{{ name }}
-              {{ assessmentStatusTag(assessment.status) | safe }}</h1>
+            <h1 class="govuk-heading-l">
+                <span class="govuk-caption-l">Referral summary</span>
```

Comment:
I don't think the change of size modifier from `-xl` to `-l` is desirable here? This does change the rendered size.

Reply:
Fixed now.